third_party/2and3/flask/app.pyi

```diff
@@ -96,8 +133,8 @@ class Flask(_PackageBoundObject):
     def make_null_session(self): ...
     def register_blueprint(self, blueprint: Blueprint, **options: Any) -> None: ...
     def iter_blueprints(self): ...
-    def add_url_rule(self, rule: str, endpoint: Optional[str] = ..., view_func: Callable[..., Any] = ..., provide_automatic_options: Optional[bool] = ..., **options: Any) -> None: ...
-    def route(self, rule: str, **options: Any) -> Callable[[Callable[..., _T]], Callable[..., _T]]: ...
+    def add_url_rule(self, rule: str, endpoint: Optional[str] = ..., view_func: _ViewFunc = ..., provide_automatic_options: Optional[bool] = ..., **options: Any) -> None: ...
+    def route(self, rule: str, **options: Any) -> Callable[[_ViewFunc], _ViewFunc]: ...
```

Comment:
Should this use a `TypeVar(..., bound=_ViewFunc)` instead?

Reply:
Makes sense, I'll push an update.